Source/Atomic/Graphics/Direct3D9/D3D9Graphics.cpp

```diff
@@ -268,6 +268,7 @@ Graphics::Graphics(Context* context) :
     instancingSupport_(false),
     sRGBSupport_(false),
     sRGBWriteSupport_(false),
+    numSinglePassPrimitives_(0),
     numPrimitives_(0),

```

Comment:
Ok for D3D9, metric gathering would also need to be added to D3D11 and OpenGL systems.  These changes are also missing ATOMIC BEGIN, ATOMIC END